transaction/core/tests/util/mod.rs

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) 2018-2022 The MobileCoin Foundation
 
+#![allow(unused)]
```

Comment:
or maybe just refer to the same github issue so that people know why its okay for things in this module to be flagged as unused? (I think i get it now)